src/app/modules/submission/submission.utils.ts

```diff
@@ -195,6 +195,15 @@ export const isAttachmentResponse = (
   )
 }
 
+/**
+ * Checks if a response is a quarantined attachment response to be processed by the virus scanner.
+ */
+export const isQuarantinedAttachmentResponse = (
+  response: ParsedClearFormFieldResponse,
+): response is ParsedClearAttachmentResponse => {
```

Comment:
/nit: to update jsdoc for ParsedClearAttachmentResponse (for both email and storage mode form)